l10n_it_fatturapa_out/__openerp__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     'name': 'Italian Localization - Fattura elettronica - Emissione',
-    'version': '8.0.3.3.2',
+    'version': '8.0.3.3.3',
```

Comment:
non vanno più inserite le modifiche di versione, è stato automatizzato con ocabot

Reply:
wow, dove?

Reply:
@SimoRubi https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92